requirements.txt

```diff
@@ -1,29 +1,29 @@
 alembic==1.0.0
 async-timeout==4.0.2
-attrs==22.2.0
+attrs==23.1.0
 Babel==2.9.1
 Beaker==1.11.0
 bleach==3.3.0
-boto3==1.26.73
-botocore==1.29.73
+boto3==1.26.115
+botocore==1.29.115
 certifi==2022.12.7
 cffi==1.15.1
 chardet==3.0.4
 ckan==2.9.8
-ckanext-datajson==0.1.15
+ckanext-datajson==0.1.18
 ckanext-dcat-usmetadata==0.3.9
-ckanext-envvars==0.0.2
+ckanext-envvars==0.0.3
```

Comment:
Wow, I didn't realize they updated this last year... https://pypi.org/project/ckanext-envvars/#history I thought it hadn't changed in 8 years.

Reply:
It was because of CKAN 2.10 breaking changes.. so yeah haha..